app/server/auth/authenticate.py

```diff
@@ -27,6 +31,9 @@ async def get_token_from_gitlab(
 
 @router.get("/github/token")
 def get_token_from_github(code: str, client_id: str, client_secret: str) -> dict:
+    """
+    Get the token from GitHub, and authorize request to the OPA Manager
```

Comment:
```suggestion
    Get the token from GitHub and authorize the request to the OPA Manager
```

This function retrieves only the token and doesn't authorize the OPA Manager application.